realm/realm-library/build.gradle

```diff
@@ -92,6 +92,9 @@ android {
     packagingOptions {
         exclude 'META-INF/NOTICE.txt'
         exclude 'META-INF/LICENSE.txt'
+        // We did strip with cmake for release build.
+        // Please, Gradle, you are not that smart! Pleas DO NOT strip debug symbols for debug build!
+        doNotStrip "*/*/*.so"
```

Comment:
I wonder why they kept this. https://google.github.io/android-gradle-dsl/current/com.android.build.gradle.internal.dsl.PackagingOptions.html#com.android.build.gradle.internal.dsl.PackagingOptions:doNotStrip isn't particular helpful on that front. Perhaps useful if not using CMake